keyboards/linworks/whale75/readme.md

```diff
@@ -0,0 +1,40 @@
+# Lx3 Whale 75
```

Comment:
Can you make this to be more in line with our [current template](https://github.com/qmk/qmk_firmware/blob/master/quantum/template/avr/readme.md)?